monitoring/soca_plotfield.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+#Command:
+#   bash soca_plotfield.sh -x exp_name -v state_vector -s start_date -e end_date
+
+function prepsurfyaml {
+cat <<EOF
+variable: $1
+clim:
+  min: $2
+  max: $3
+time: $4
+level: $5
+latitude: 0.0
+aggregate: False
+projection: $6 #'global'
+experiment: $7 
+plot_dir: $8
+color: $9  # Spectral  #seismic #jet 
+EOF
+}
+
+while getopts x:v:s:e:p: flag
+do
+    case "${flag}" in
+        x) exp=${OPTARG};;  # experiment
+        v) stv=${OPTARG};;  #state vector
+        p) cycle_dir=${OPTARG};;  #state vector
+        s) start_date=${OPTARG}Z00;;
+        e) end_date=${OPTARG}00;;
+    esac
+done
+echo experiments: $exp
+fctl=24
+#cycle_dir=/work/noaa/marine/jhossen/$exp
+echo 'EXP dir:' ${cycle_dir}
+cdir=$PWD/$exp
```

Comment:
Remove $PWD. It is the directory where the scripts reside. Should be exppath

Reply:
It should not be. $PWD is for the plotting dir.